api/src/db/seeds/test/campaigns.js

```diff
@@ -13,7 +13,7 @@ exports.seed = async (knex) => {
     dirname: path.join(__dirname, '..', '..', '..', '..', 'tests', 'tapes')
   }))
 
-  const tm4proxy = http.createServer(yakbak('https://tasks-backend.openstreetmap.us/', {
+  const tm4proxy = http.createServer(yakbak('https://tasks-backend.openstreetmap.us/api', {
```

Comment:
@kamicut I see a response for the original URL, but a 404 for this change.

Reply:
Yeah, it's only used in the construction of API urls as a prefix, for example https://tasks-backend.openstreetmap.us/api/v2/projects